src/common/tiglcommonfunctions.cpp

```diff
@@ -1214,7 +1215,11 @@ TopoDS_Shape SplitShape(const TopoDS_Shape& src, const TopoDS_Shape& tool)
         LOG(ERROR) << ss.str();
         throw tigl::CTiglError(ss.str());
     }
+#if OCC_VERSION_HEX >= VERSION_HEX_CODE(7,2,0)
```

Comment:
Why these changes? We are using our own Geom_AlgoSplitter, right? Or do we now the one shipping with OCCT 7.2.0?

Reply:
yes, you are using your own. but your own inherits from OCC's BOPAlgo_Builder and the methods related to errors have been reworked in 7.2